examples/saul/Makefile

```diff
@@ -9,6 +9,9 @@ RIOTBASE ?= $(CURDIR)/../..
 
 # we want to use SAUL:
 USEMODULE += saul_default
+
+USEMODULE += lis2dh12
```

Comment:
I don't think upstream RIOT administrators will allow you to add the LIS2DH12 as dependency to the general saul example. Please remove these changes.